nncf/experimental/torch/fx/transformations.py

```diff
@@ -136,17 +142,41 @@ def bias_update_transformation(model: torch.fx.GraphModule):
     return bias_update_transformation
 
 
-def constant_update_transformation_builder(node: NNCFNode, value: torch.Tensor) -> TransformationFNType:
+def shared_constants_unification_transformation(model: torch.fx.GraphModule):
+    """
+    checks fx graph for shared constants, disconnects and eliminates redundant
+    shared constant while connecting singular shared constant.
```

Comment:
Please elaborate a little more which problem this function solves.
From the current description it's not clear why some nodes should be disconnected and eliminated. 
Maybe it's worth mentioning the issue with `solver` and `min_max` algorithms that they don't use `is_shared` attribute and so on.

Reply:
Alright, I will elaborate the comment.